### ISSUE TITLE
Meshroom 2019.2.0 for linux requires Cuda 3.0 not 2.0 !!!

### ISSUE THREAD

**Issue by cptX** [alicevision/Meshroom#734]
Hi, according to the release description Meshroom 2019.2.0 should require Cuda >= 2.0 but when running Feature Extraction reports 

[02:06:25.958896][error] CUDA-Enabled GPU detected, but the compute capabilities is not enough.
- Device 0: 2.1, global memory: 961MB
 - Requirements: 3.0, global memory: 0MB

[02:06:25.958939][info] CUDA-Enabled GPU not supported.

(See attached screenshot)
![Screenshot from 2019-12-14 03-02-01](https://user-images.githubusercontent.com/9040042/70840956-ef3fa800-1e1e-11ea-8566-3e3a74801df6.png)

My card is Nvidia GTS 450 with Cuda 2.1 capabilities

What is wrong here?

**Comment by natowi** [comment]
Yes, that´s because https://github.com/alicevision/popsift used in feature extraction needs cuda v3.
Feature extraction with gpu is experimental and disabled by default.

**Comment by cptX** [comment]
It is working so probably it forces CPU by itself. My question is what version of Meshroom can I use to take advantage of my cuda 2.1?

**Comment by natowi** [comment]
You can use the current version of Meshroom, but  Feature extraction will be computed by your cpu (do not disable "fore cpu extraction" as gpu computation requires c.c.3.0). The DephMap node will make use of your cuda gpu with compute capability 2.1

**Comment by exilaus** [comment]
Ok i try and same result ok if need c.c.3.0 need find new gpu. Anyway tomorrow will continue process and try if deep map can use c.c.2.
![20200316_012147](https://user-images.githubusercontent.com/2852823/76714026-05624a80-6725-11ea-8c9f-540c25bd541f.jpg)
